pkg/branding/brandedlogo_test.go

```diff
@@ -0,0 +1,76 @@
+package branding
+
+import (
+	"embed"
+	"encoding/base64"
+	"testing"
+
+	"github.com/stackrox/rox/pkg/testutils/envisolator"
+	"github.com/stackrox/rox/pkg/utils"
+	"github.com/stretchr/testify/suite"
+)
+
+var (
+	//go:embed files/red_hat_acs_logo_rgb.png
+	logoRHACStest       embed.FS
+	logoRHACSBase64test = func() string {
```

Comment:
I don't know if embedding the same files the same way in the test as in tested code makes the test that useful. I wonder if you can get by simply hardcoding first few characters of base64 string of file contents or a checksum of base64 string.
This will at least prove that:
* files were read
* files were not empty
* RHACS and StackRox logos are different
* files are expected ones, if you use checksum for validation

WDYT?

Reply:
Sounds reasonable, I'll do it.